chalice/app.py

```diff
@@ -200,6 +203,10 @@ def maybe_fake_504() -> typing.Optional[chalice.Response]:
         res_headers.pop("Content-Length", None)
         res_headers["Strict-Transport-Security"] = "max-age=31536000; includeSubDomains; preload"
         res_headers["X-AWS-REQUEST-ID"] = app.lambda_context.aws_request_id
+
+        if include_retry_after_header(return_code=status_code, method=app.current_request.method, uri=path):
```

Comment:
This appears to attach a `Retry-After` header to all responses except those listed in `include_retry_after_header`, including 200 responses.

Reply:
sigh

Reply:
Will fix.